command-line-downloader/src/main/java/uk/ac/ebi/ena/app/utils/MenuUtils.java

```diff
@@ -45,12 +43,16 @@
     public static final String welcomeMessage = "Welcome to the Ena file downloader utility!";
     public static final String lineMessage = "----------------------------------------------";
     public static final String accessionsErrorMessage = "Please provide valid accessions! ";
+    public static final String validAccessionsErrorMessage = "Please provide accessions in valid format. A valid" +
+            "accessions is one which is generated by portal api or  coviddataportal! ";
```

Comment:
Accessions file should be plain-text, in TSV (tab separated values) format. The first column must be the accessions. Header row is ignored if present. Values can be enclosed in double quotes or not.